keyboards/angel64/info.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
You can actually delete this file completely. The `info.json` files in `keyboards/angel64/alpha/` and `keyboards/angel64/rev1/` supersede it.